app/script/view_model/WindowTitleViewModel.js

```diff
@@ -54,9 +54,17 @@ z.viewModel.WindowTitleViewModel = class WindowTitleViewModel {
         const unreadConversations = this.conversationRepository
           .conversations_unarchived()
           .filter(conversationEntity => {
-            const isIgnored = conversationEntity.is_request() || !conversationEntity.showNotificationsEverything();
-            const hasActions = conversationEntity.unreadMessagesCount() || conversationEntity.hasJoinableCall();
-            return hasActions && !isIgnored;
+            const {unreadMessages, unreadSelfMentions} = conversationEntity.unreadState();
+
+            const isIgnored = conversationEntity.is_request() || conversationEntity.showNotificationsNothing();
```

Comment:
There might be a logical issue here 
Those two are not equivalent:

`!conversationEntity.showNotificationsEverything()`
`conversationEntity.showNotificationsNothing()`

Reply:
Maybe this is an intended change?

Reply:
This is an intended change to return early if all notifications are disabled. Correct me if I'm wrong but I see that as logical.